### ISSUE TITLE
Sending ONE from 1Wallet

### ISSUE THREAD

**Issue by HarmoniusDude** [polymorpher/one-wallet#239]
15000ms fail, transaction won't finalize, transaction successful without ONE being sent to Harmony Wallet. When I go to send and hit max, a zero is entered even with Harmony in the 1wallet. Can't send funds.

**Comment by polymorpher** [comment]
Were you hitting daily limit? Also it would help if you post a transaction / explorer link.

**Comment by HarmoniusDude** [comment]
I believe that was the issue. Later last night I was able to send and it worked perfectly.

**Comment by HarmoniusDude** [comment]
Yeah, I think that would have cleared up the confusion, but I clicked on a few tabs and saw the 1,000/daily limit and figured I just needed to wait. I appreciate getting back to me and love the new wallet. Works so fast and really straight forward.
